style.css

```diff
@@ -0,0 +1,218 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 12px;
+    background-color: transparent;
+    min-width: 1024px;
+    min-height: 830px;
+    max-height: 1472px;
+    margin-top: 0 !important;
+    padding-top: 0 !important;
+    display: block;
+    justify-content: center;
+    text-align: center;
+    max-width: fit-content;
+}
+
+.new-products {
+    display: block;
+    position: relative;
+    text-align: center;
+    width: 100%;
+}
+
+.new-products > .products {
+    display: inline-block;
+    width: 24.8%;
+}
+
+.products img {
+    position: relative;
+    min-width: 25%;
+    left: 0;
+    width: 100%;
+    height: 210px;
+    border: 2px;
+    padding: 0.5px;
+}
+
+.footer-content {
+    font-family: 'Roboto', sans-serif;
+    background: white;
+}
+
+.footer-content a {
+    text-decoration: none;
+    color: #348dce;
+}
+
+.footer-content > .spacer {
+    line-height: 12px;
+    padding-left: 10px;
+}
+
+.main_nav {
+    color: white;
+    display: block;
+    position: relative;
```

Comment:
#### `REQUIRED`

### :bug: This is what we need to change to fix the unwanted gutter at the top of the page. :smile: 

![gutter-apple-unwanted](https://user-images.githubusercontent.com/36057474/64074706-db749980-cca6-11e9-8939-479da98923ec.png)

```suggestion
    position: absolute;
    left: <any length value>;
```

#### See demo below:

![fix-apple-bug](https://user-images.githubusercontent.com/36057474/64074772-9ac95000-cca7-11e9-9193-f61170b37e55.gif)

### Learn more about [**positioned elements** here](https://developer.mozilla.org/en-US/docs/Web/CSS/position)